init.lua

```diff
@@ -60,6 +60,29 @@ local source = string.sub(debug.getinfo(1,'S').source, 2)
 local path = string.sub(source, 1, string.find(source, "/[^/]*$"))
 local noicon = path .. "noicon.png"
 
+local function cycle(altTabTable, altTabIndex, dir)
+   -- Switch to next client
+   altTabIndex = altTabIndex + dir
+   while altTabIndex > #altTabTable do
```

Comment:
The orignal "wrap around" only works if `dir = 1` or `dir = -1`). As this is not the case when arbitrary clients are selected with the mouse, I implemented a proper wrap around (similar to modular arithmetic, but off by 1 due to Lua indices starting at 1). A `if` instead of the `while` loop should be enough as well for the moment, but to be honest, the while loop doesn't hurt here (and makes the implementation more generic for further features where `dir` maybe gets very big).